web_client/analysis-modules/base-listing.jsx

```diff
@@ -3,13 +3,15 @@ import igpse from './igpse';
 import kmeans from './kmeans';
 import silhouette from './silhouette';
 import surv from './surv';
+import featureMatching from './feature-matching';
 
 /* eslint-disable */
 export default [
-  { key: 'igpse'     , name: 'iGPSe'          , module: igpse      },
-  { key: 'kmeans'    , name: 'K Means'        , module: kmeans     },
-  { key: 'silhouette', name: 'Silhouette Plot', module: silhouette },
-  { key: 'surv'      , name: 'Survival Plot'  , module: surv       }
+  { key: 'igpse'     , name: 'iGPSe'           , module: igpse           },
```

Comment:
I think there's an argument to be made similar to my note above about PEP8, but in this case, it seems to me that this code can take on many possible formats and still be readable, so I will go ahead and fix this up.  Please let me know what you think of the result.I think there's an argument to be made similar to my note above about PEP8, but in this case, it seems to me that this code can take on many possible formats and still be readable, so I will go ahead and fix this up.  Please let me know what you think of the result.